### PR TITLE
Updated list items causing errors during the `npm test` run

### PR DESCRIPTION
**Pull request by Manvel**
- This PR addresses the errors caused by outdated list.

### CODE REVIEW

**Comment by Manvel** [comment]
- There is still the generator error though
```
FAILURE: 561/562 tests ran successfully; 1 failed, 0 incomplete, 1 errors

Error #1:
static site generators list ➞ local render ➞ render local data
TypeError: Cannot read property 'message' of null
    at /home/saroyanm/projects/websites/static-list/node_modules/getrepos/source/index.js:244:27
    at TaskGroup.<anonymous> (/home/saroyanm/projects/websites/static-list/node_modules/feedr/source/index.js:532:5)
    at Object.onceWrapper (events.js:317:30)
    at emitTwo (events.js:126:13)
    at TaskGroup.emit (events.js:214:7)
    at TaskGroup.listener (/home/saroyanm/projects/websites/static-list/node_modules/taskgroup/source/lib/interface.js:59:10)
    at emitTwo (events.js:126:13)
    at TaskGroup.emit (events.js:214:7)
    at TaskGroup.finish (/home/saroyanm/projects/websites/static-list/node_modules/taskgroup/source/lib/taskgroup.js:976:8)
    at TaskGroup.fire (/home/saroyanm/projects/websites/static-list/node_modules/taskgroup/source/lib/taskgroup.js:1041:10)
```

I'm not sure how to fix this..

File: list.json [hunk 1]
```diff
@@ -2050,10 +2049,6 @@
     "github": "skx/templer",
     "license": "Artistic-1.0-Perl"
   },
-  {
-    "name": "Tempo",
-    "github": "catnapgames/Tempo"
```
Comment (Manvel) [inline list.json]: The repository doesn't exist at all.

File: list.json [hunk 2]
```diff
@@ -2175,13 +2170,6 @@
     "name": "Webhook",
     "github": "webhook/webhook"
   },
-  {
-    "name": "Website Meta Language",
```
Comment (Manvel) [inline list.json]: The project website is not available anymore.

Comment (Manvel) [reply list.json]: Considering that the last release was in 2006, I assume the project doesn't exist anymore.

**Comment by balupton** [comment]
Great thanks. I'll look into the dependency failure.

**Comment by balupton** [comment]
Added you to the maintainers team.

**Comment by Manvel** [comment]
@balupton Any findings regarding the failure ? As the website is not updated yet I assume there is still a tests error that prevents the website from being updated.

**Comment by balupton** [comment]
That failure pops up when you hit the github rate limits. There should be an earlier warning or so about it.

**Pull request merged**